aws/terraform/README.md

```diff
@@ -30,9 +15,34 @@ See the documentation for the [AWS provider](https://registry.terraform.io/provi
 
 ### Required Tools
 
-This module depend on other utilities for access to remote resources. The following are required:
+These modules depend on other utilities for access to remote resources. The following are required:
 
 - [AWS CLI](https://aws.amazon.com/cli/), logged in to an account with access to the EKS cluster
 - [kubectl](https://kubernetes.io/docs/reference/kubectl/), with a [properly configured kubeconfig file](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html), allowing access to the EKS cluster
 
 [^1]: Run `kubectl config get-contexts` to see a list of all contexts in your kubeconfig file; if `config_context` is not set in `providers.tf`, it will default to your current Kubernetes context, as long as one is set in your kubeconfig file
+
+## IAMServiceAccounts
+
+This module creates IAM Roles and Policies, as well as Kubernetes ServiceAccounts, to allow certain workloads access to AWS resources, such as Route 53 and S3. See the [module README](modules/iamserviceaccounts/README.md) for details.
+
+### Required Values for IAMServiceAccounts
+
+The following values must be set when invoking this module from `main.tf`. See the [module README](modules/iamserviceaccounts/README.md) for the full list of possible variables.
+
+- `dns_zone_name` = the subdomain for a new DNS zone, the module will attempt to create this DNS Zone in Route 53
+- `eks_cluster_name` = the name of an existing EKS cluster
+- `oidc_provider_url` = the URL of an existing OpenID Connect (OIDC) Provider ([see the AWS documentation](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html) to enable an OIDC Provider for your cluster)
+- `s3_bucket` = the name for a new S3 Bucket to hold application database backups; the module will attempt to create this Bucket[^2]
+
+[^2]: Remember that S3 Bucket names must be globally unique
+
+## Elastic File System Module
+
+This module creates a Security Group in your cluster's VPC to allow NFS traffic on port 2049. It then creates an Elastic File System and Mount Targets for that file system in each subnet associated with your cluster. Finally, it creates an EFS-backed StorageClass on your cluster. See the [module README](modules/elastic-file-system/README.md) for details.
```

Comment:
Should the EFS CSI driver be installed before running this module? It would be good make that clear in the doc.

Reply:
Good point. I've added that as a note here. It may be a good idea soon to write a module that installs EKS add-ons as well. I've looked a bit at the docs and it doesn't seem very hard.